Samples/london-cycle-traffic-air-quality-lsoa-backoff.py

```diff
@@ -1,34 +1,60 @@
+'''
+  This recipe gets two datasources
+     - London Traffic data using TrafficCountImporter.java
+     - London localAuthority data using OaImporter.java
+  
+  By using ArithmeticField and LatestValueField if perform two operations
+    - Sum of the mean of 'CarCountTaxis' - 1
+    - Sum of the mean of 'CountPedalCycles' - 2
+  
+  Then for each localAuthority using GeographicAggregationField
+    - It dividies 2/1 to get 'BicycleFraction' - 3
+  
+  Then writes 3 to the output file for every localAuthority using GeographicAggregationField
```

Comment:
Same as before: we can say it then sums the resulting field...

Reply:
done